test/WebSites/VersioningWebSite/VersioningWebSite.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;netcoreapp1.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp1.1</TargetFrameworks>
```

Comment:
🙈 not much point in compiling these and it adds at least 10s to a Linux build in my testing. Probably more since I only compared `build-compile` runs.